### DISCUSSION TITLE
Successfully install from bash scrip in AWS but WireGuard does not start up

### DISCUSSION THREAD

**Discussion by sanskritkm** [pivpn/pivpn#1530] [Support]
## In raising this issue, I confirm the following:

`{please fill the checkboxes, e.g: [X]}`

- [x] I have read the [documentation](https://docs.pivpn.io)
- [x] Is it a feature request? please consider opening a [Discussion] (https://github.com/pivpn/pivpn/discussions/new)
- [x] I have read and understood the [contributors guide](https://github.com/pivpn/pivpn/blob/master/CONTRIBUTING.md).
- [x] The issue I am reporting can be *replicated*.  
- [x] The issue I am reporting *is* directly related to the pivpn installer script.
- [x] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pivpn/pivpn/wiki/FAQ), [closed issues](https://github.com/pivpn/pivpn/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aclosed), and [open issues](https://github.com/pivpn/pivpn/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aopen)).

### Has your install failed? 
```
no
```

### Describe the issue

I install PiVPN in AWS with successful but can't start 

**Job for wg-quick@wg0.service failed because the control process exited with error code.**

### Can you replicate the issue? Describe the steps below

1. Go to **_curl -L https://install.pivpn.io | bash_**
2. Success
3. Auto reboot

### Have you searched for similar issues and solutions?
    (yes/ which issues?)

```
https://github.com/pivpn/pivpn/issues/1175

 but no help
```

**Additional context**

### Have you taken any steps towards solving your issue?
```
  which? 
sudo dpkg-reconfigure wireguard-dkms

```

#### What type of hardware are you running PiVPN at?

```
AWS

```
#### Output of `uname -a`
```
  Linux ip-172-31-30-248 4.19.0-16-cloud-amd64 #1 SMP Debian 4.19.181-1 (2021-03-19) x86_64 GNU/Linux

```

#### Output of `cat /etc/os-release`
```
 PRETTY_NAME="Debian GNU/Linux 10 (buster)"
NAME="Debian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"

```

### If install failed Please provide the console output of  `curl -L https://install.pivpn.io | bash`
```
 https://pastebin.com/gCB69cBa
```

### Console output of      `curl -L install.pivpn.io | bash`
```
 https://pastebin.com/SUP07ajF

```

### Console output of      `pivpn debug`

```
  https://pastebin.com/AbqqscsL

```

**Comment by orazioedoardo** [comment]
Make sure you have the latest kernel (which version?). If you don't, update and reboot. Also you can just post the output here instead of adding a layer of indirection via pastebin.

**Comment by coolapso** [comment]
which AMI are you using? (can i get AMI ID and Region?) 
have you done `sudo apt update && sudo apt upgrade`  before running pivpn? 

Are you running on user data or installing manually? have you checked this: https://docs.pivpn.io/guides/user-data/ ? 


Moving to discussions ...
